Themis/Views/Assessment/Modeling Exercise/Diagram Renderers/Use Case Diagram/UMLUseCaseDiagramRenderer.swift

```diff
@@ -8,14 +8,19 @@
 import SwiftUI
 
 struct UMLUseCaseDiagramRenderer: UMLDiagramRenderer {
-    var context: GraphicsContext
+    var context: UMLGraphicsContext
     let canvasBounds: CGRect
+    var fontSize: CGFloat
+    
+    init(context: GraphicsContext, canvasBounds: CGRect, fontSize: CGFloat = 14) {
+        self.context = UMLGraphicsContext(context)
+        self.canvasBounds = canvasBounds
+        self.fontSize = fontSize
+    }
```

Comment:
Same as for the class diagram renderer.
```suggestion
    }
```